### ISSUE TITLE
Does not work with stylus-loader 4.x

### ISSUE THREAD

**Issue by zaneclaes** [naftis/craco-stylus#1]
I just found this repo today, and it's exactly what I need. However, I could not get it to work until I downgraded `stylus-loader` to `3.x`.

Specifically:

```
node_modules/stylus-loader/dist/cjs.js??ref--5-oneOf-8-3!./src/styles/app.styl)
ValidationError: Invalid options object. Stylus Loader has been initialized using an options object that does not match the API schema.
 - options has an unknown property 'source'. These properties are valid:
   object { stylusOptions?, sourceMap?, webpackImporter?, additionalData? }
```

I ended up copying the source code and editing it appropriately for the new docs: https://webpack.js.org/loaders/stylus-loader/

**Comment by naftis** [comment]
Hi @zaneclaes, thanks for reporting the bug. If you modified the source code, would you mind creating a pull request for fixing this? My project where I'm using this library is locked with the older Stylus, so I'm not currently using the latest versions unfortunately.

**Comment by TheSixer** [comment]
Hi @zaneclaes , did you fixed this ?

**Comment by TheSixer** [comment]
@naftis I'm sorry to copied you code, and published a new plugin for myself. I just want try fixing this bug to myself. I'm so sorry about that.

**Comment by naftis** [comment]
@TheSixer No problem, great that you got it working. I can also create a fix for this repository from your plugin if that is okay with you :)

**Comment by naftis** [comment]
Hello @TheSixer @zaneclaes, released 1.1.1 with a fix to this problem. Could you verify the issue is fixed for you?

**Comment by nikitaGetman** [comment]
Hi guys.I fixed it in my PR -  https://github.com/naftis/craco-stylus/pull/4

**Comment by naftis** [comment]
@nikitaGetman I've released 1.2.0 with your fix to this problem.
